cluster/src/main/scala/io/snappydata/cluster/ExecutorInitiator.scala

```diff
@@ -45,7 +45,7 @@ import org.apache.spark.{Logging, SparkCallbacks, SparkConf, SparkEnv}
  */
 object ExecutorInitiator extends Logging {
 
-  val SNAPPY_MEMORY_MANAGER = "org.apache.spark.memory.SnappyStaticMemoryManager"
+  val SNAPPY_MEMORY_MANAGER = "org.apache.spark.memory.SnappyUnifiedMemoryManager"
```

Comment:
ExecutorInitiator will always set memory manager to SnappyUnifiedMemoryManager? Do we want this configurable with default being SnappyUnifiedMemoryManager?